docs/installation/index.rst

```diff
@@ -0,0 +1,46 @@
+.. _installation:
+
+Installation
+============
+
+mlir-graphblas is implemented via Python and Cython.
+
+Building from source is possible, but the recommended method to install is via ``conda``.
+
+It is assumed by mlir-graphblas that it is installed using ``conda`` or ``venv``. Thus, installing using one of those two methods is recommended.
```

Comment:
This line is redundant with the one above it. And users don't install using venv, they install using pip into a venv-managed environment (as I understand it). So it would be better to simply say:

Building from source or installing from pip is possible, but the recommended method to install is using ``conda``.

Reply:
https://github.com/metagraph-dev/mlir-graphblas/pull/26/commits/46b00a45f1152e319f2d3fd7d10134e6ee16172a makes it say:

> Building from source or installing from `pip` is possible, but the recommended method to install is using `conda`.